src/Http/Parser.php

```diff
@@ -53,12 +54,7 @@ public function __construct(string $content = '')
             $decode = json_decode($content, true, 512, JSON_THROW_ON_ERROR);
 
             $this->decode = collect($decode);
-            $this->batching = $this->decode
-                ->keys()
-                ->filter(fn ($value) => is_string($value))
-                ->isEmpty();
-
-            $this->batching = $this->decode->isEmpty() ? false : $this->batching;
+            $this->batching = !$this->decode->isEmpty() && !$this->isAssociative($this->decode->toArray());
```

Comment:
I think we don't have to create a different method at all. We could also use the reverse method since we use negation everywhere. For instance: `Arr::isList(...)` Let's change it with your changes

Reply:
I will if I understand correctly

Reply:
@tabuna Done.